index.html

```diff
@@ -96,17 +113,17 @@ <h2 id="project_tagline"></h2>
 
 <p><strong>Publish on a given host, given exchange, given domain and given user domain suffix:</strong></p>
 
-<pre><code> $ java -jar publish.war -f publishMessages.json -en rabbit -mb mb101 -domain eiffel001 -ud fem001 -mp eiffelsemantics
+<pre><code> $ java -jar remrem-publish-cli.jar -f publishMessages.json -en rabbit -mb mb101 -domain eiffel001 -ud fem001 -mp eiffelsemantics
 </code></pre>
 
 <p><strong>If you want to have the message non persistent add np flag:</strong></p>
 
-<pre><code> $ java -jar publish.war -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
+<pre><code> $ java -jar remrem-publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
```

Comment:
I think the jar name mentioned here should be something like remrem-publish-cli-<version>.jar